x/auth/ante/expected_keepers.go

```diff
@@ -19,6 +20,13 @@ type AccountKeeper interface {
 	AddressCodec() address.Codec
 }
 
+// UnorderedSequenceManager defines the contract needed for UnorderedSequence management.
+type UnorderedSequenceManager interface {
```

Comment:
How about `UnorderedNonce`? It's not really a sequence

Reply:
makes sense, updated to nonce